website/docs/intro.md

```diff
@@ -3,30 +3,34 @@ sidebar_position: 1
 title: "Contributor Guide"
 ---
 
-
 ## We ♥️ Contributions!
-
 `awesome-azd` is a community-friendly resource to help developer discover, create, and share, _azd-template_ resources for streamlining your application development and CI/CD pipelines. It is an _open-source_ effort open for contributions in many ways. 
 
-
 ## Our Custom Issues
-
 The [custom issues](https://github.com/Azure/awesome-azd/issues/new/choose) page shows the different ways in which _anyone_ can contribute to make this project successful for _everyone_. 
-Here are the 4 main ones:
 
- - submit a **resource** for the `awesome-azd/README.md` list
- - submit an **idea** for a template
- - contribute a **requested template** matching an idea
- - contribute a **new template** from your own project or idea
+### Submit a Template Idea
+There are many ways to combine languages with Azure services and tools to create applications, so if you cannot find a template with architecture that works for you-- you can submit a request for that template.
+
+Keep in mind, a template are made to be adaptable. You can use a template's architecture and swap out the source code. For example, if you want to create a grocery list making applciation using Azure SQL and Azure App service, you can use the [ToDo Application in C# with Azure SQL and App Service](https://github.com/Azure-Samples/todo-csharp-sql) template and swap out the source code. 
 
-![](./../static/img/custom-issues.png)
- 
+### Contribute a Template
+Our template gallery will be at its best when our community is involved in creating and contributing new templates. 
+
+We welcome contributions of [requested templates](https://github.com/Azure/awesome-azd/issues?q=is%3Aopen+is%3Aissue+label%3Arequested-contribution) as well as any others, subject to review. 
+
+If you would like to contribute a template but are not sure where to start, [making an existing project azd compatible](https://learn.microsoft.com/en-us/azure/developer/azure-developer-cli/make-azd-compatible) is a great first step. Doing so consists of three main steps:
+1.  Set up a new environment (with `azd init`)
+2.  Add Bicep files
+3.  Update azure.yaml 
+
+### Submit a Resource
+Did you write or find an article that helped you get started with `azd`? Or maybe you created or found a video that showed you how to create an azd template? Whatever the resource might be, we would love for you to share it with our community! So, use this issue to submit content you think should be included in `awesome-azd/README.md`
 
 ## Other Ways To Help 
-Other than these, we always welcome feedback in terms of:
+Other than these, we always welcome feedback through:
  - **bug reports**: let us know if something is broken
```

Comment:
Perhaps the bullet points should include direct links as well (bug report & feature request).